src/util/SignalsHandler.cpp

```diff
@@ -119,10 +119,8 @@ SignalsHandler::~SignalsHandler()
 void
```

Comment:
For some reason I can't leave the comment at any place in the file.

If this is fixing #1379, we need to create a new issue to move bool in callback into stop token and adjust this TODO.
https://github.com/XRPLF/clio/blob/0dcbbf9afa9bd8b95d173cd3b0a4b669fb3d52bb/src/util/SignalsHandler.cpp#L86

Reply:
I'm having second thoughts about changing it this way. But i'll create the ticket and fix this todo. We can then decide later what to do with it 👍